listview-fullscreen/app/src/main/res/layout/list_item.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="16dp"
+    android:paddingRight="16dp"
+    android:paddingTop="8dp"
+    android:paddingBottom="8dp">
+
+    <com.longtailvideo.jwplayer.JWPlayerView
```

Comment:
@paulmandal 
We could do an extra optimization here, by showing an `ImageView` on default and replacing it with a JWPlayerView in `onClick`. This will prevent quite a few expensive JWPlayerView instantations.